src/main/kotlin/oop/Facade/Facade.kt

```diff
@@ -0,0 +1,24 @@
+package oop.Facade
+
+class NetInvoiceSalaryFacade(val iva: IVAOperation,
+                             val irpf: IRPFOperation) {
+
+  fun calculate(salary: Double) : Double =
+    salary + iva.apply(salary) - irpf.apply(salary)
+
+  fun calculateAnnual(monthlySalary: Double) : Double =
+    calculate(monthlySalary * 12)
+
+}
+
+class IVAOperation {
```

Comment:
Different things, this is only a example of Facade.

Has not relationship with little thai